library/ziti_ctrl.c

```diff
@@ -430,6 +430,9 @@ static void ctrl_body_cb(tlsuv_http_req_t *req, char *b, ssize_t len) {
     if (len > 0) {
         if (resp->resp_content == ctrl_content_json) {
             if (resp->content == NULL) {
+#ifdef ZITI_DEBUG
+                CTRL_LOG(DEBUG, "HTTP RESPONSE: %s", b);
+#endif
```

Comment:
you don't need this guard, and log at VERBOSE level
also `b` is not guaranteed to be `\0` terminated
you need to use `%.*s` formatting